src/wallet/rpc/spend.cpp

```diff
@@ -418,7 +418,8 @@ RPCHelpMan sendmany()
 RPCHelpMan settxfee()
 {
     return RPCHelpMan{"settxfee",
-                "\nSet the transaction fee rate in " + CURRENCY_UNIT + "/kvB for this wallet. Overrides the global -paytxfee command line parameter.\n"
+                "\nWill be deprecated see 'settxfeerate'.\n"
```

Comment:
Why "will be deprecated"? Why not actually deprecate it?

Reply:
Avoid incompatibility problems with programs actually using this RPC call.

Reply:
That's what we have the `-deprecatedrpc` parameter for

Reply:
I will mark this as resolved as we're discussing the same in the other comment thread